.github/workflows/pr.yml

```diff
@@ -0,0 +1,54 @@
+name: pr
+
+on: pull_request
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  type:
+    name: ʦ Type check
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: lts/*
+      - name: Install dependencies
+        run: npm ci
```

Comment:
Merge here the biome:ci script

Reply:
And then you can also trigger a bulld step.

To recap, have one job to:

1. Install deps
2. Run biome:ci
3. Build the app